packages/node_modules/@webex/plugin-meetings/test/integration/spec/journey.js

```diff
@@ -286,6 +297,80 @@ skipInNode(describe)('plugin-meetings', () => {
           throw e;
         }));
 
+      // FHD - remote video tests - starts
+      // Firefox bug - https://bugzilla.mozilla.org/show_bug.cgi?id=1537986
+      if (getBrowserName() !== 'Firefox') {
+        const getRemoteHeight = (minHeight = 360, maxHeight = 1080) => new Promise((resolve, reject) => {
+          testInterval = setInterval(() => {
+            const remoteSettings = bob.meeting.mediaProperties.remoteVideoTrack.getSettings();
+            const height = remoteSettings.height && +remoteSettings.height;
+
+            console.log(`Resolution for ${minHeight} ${maxHeight} : `, count, height);
+            if (height >= minHeight && height <= maxHeight) {
```

Comment:
maybe it would be better to change this to check to > and < so that then the arguments passed to getRemoteHeight() could be the same as later used in the asserts. Currently it looks a bit silly and messy that you have to manually add or remove 1 to the values when using the asserts and in many places you're actually using completely different values in the asserts which seems like a mistake, or if it's not a mistake then there should be comments explaining why the values are different

Reply:
Even if we use > and <, assert range will differ as mentioned [here](https://github.com/webex/webex-js-sdk/pull/2446#discussion_r986433243). Adding a comment to explain why the values are different.